### PR TITLE
Implemented jump to future or past with index. closes #13

### LINKED ISSUE #13: allow seeking to specific state in history
Just noticed it when messing with debugging. You are passing number of steps to undo/redo functions, but they are not handled in there.

### PR DESCRIPTION
**Pull request by leomelin**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -41,6 +43,12 @@ export const ActionCreators = {
   },
   redo () {
     return { type: ActionTypes.REDO }
+  },
+  jumpToFuture (index) {
+    return { type: ActionTypes.JUMP_TO_FUTURE, index: index }
```
Comment (omnidan) [inline src/index.js]: can you use `index` instead of `index: index` here too? (like below, just for consitency)

**Comment by omnidan** [comment]
@leomelin Thanks a lot! :grin: Please fix the small inconsistency and I'll merge this :smile:

**Comment by leomelin** [comment]
Fixed!

**Comment by omnidan** [comment]
Thank you @leomelin :ok_hand:

**Comment by omnidan** [comment]
@leomelin oh I totally forgot, could you update the README.md too? I'll release a new version then :grin:

**Pull request merged**